libmscore/sym.cpp

```diff
@@ -2682,10 +2682,12 @@ const std::array<const char*, int(SymId::lastSym) + 1> Sym::symUserNames = { {
     QT_TRANSLATE_NOOP("symUserNames", "No symbol"),
     "4-string tab clef",
     "6-string tab clef",
-    "11 large diesis down, 3\u00b0 down [46 EDO]",
-    "11 large diesis up, (11L), (sharp less 11M), 3\u00b0 up [46 EDO]",
-    "11 medium diesis down, 1\u00b0[17 31] 2\u00b046 down, 1/4-tone down",
-    "11 medium diesis up, (11M), 1\u00b0[17 31] 2\u00b046 up, 1/4-tone up",
+    QT_TRANSLATE_NOOP("symUserNames", "11 large diesis down, 3\u00b0 down [46 EDO]"),
+    QT_TRANSLATE_NOOP("symUserNames", "11 large diesis up, (11L), (sharp less 11M), 3\u00b0 up [46 EDO]"),
+    //QT_TRANSLATE_NOOP("symUserNames", "11 medium diesis down, 1\u00b0[17 31] 2\u00b046 down, 1/4-tone down"),
```

Comment:
Do we still need this?

Reply:
That is the original string as taken from SMuFL, but as such doesn't work with the translations system. So I'd rather keep this (and commented out), as a hint should this string ever change

Reply:
Hmm, maybe not...
We'll see once translations got enabled. I do remember there are problems with that `\u` notation and the translations system, if it is not really clear where the number stops and the next 'normal' letter shows